core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
It is > 50 times called in total. Whereas now > 100 times in total as called twice in that method. So would save 50 times. Again, it isn't a huge improvement but it was faster this way and plugins could overwrite the value if needed. Also it was nicer to use. If it is not good for DI then I reckon it can be removed.